sdk/go/Makefile

```diff
@@ -14,18 +14,23 @@ include ../../build/common.mk
 # `test_all` without the dependencies.
 TEST_ALL_DEPS ?= install
 
-gen::
+.PHONY: gen
+gen:
 	go generate ./pulumi/...
 
 ensure: .make/ensure/go
 
-build:: gen
-	go install -C pulumi-language-go \
-		-ldflags "-X github.com/pulumi/pulumi/sdk/v3/go/common/version.Version=${VERSION}" ${LANGHOST_PKG}
+build:: gen ../../bin/pulumi-language-go
```

Comment:
Do we want to make a similar change for nodejs and python?

Reply:
Yes.